src/components/ControlledInputField/ControlledInputField.js

```diff
@@ -0,0 +1,60 @@
+import { makeStyles, TextField } from "@material-ui/core"
+import { Controller } from "react-hook-form"
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    display: "flex",
+    flexDirection: "column",
+    fontSize: theme.typography.fontSize,
+    fontFamily: theme.typography.fontFamily,
+
+    "& .MuiFormHelperText-contained": {
+      margin: "0",
+    },
+
+    "& .MuiOutlinedInput-input": {
+      padding: "6px 10px 6px 10px",
+      fontSize: theme.typography.fontSize,
+    },
+  },
+  label: {
+    fontWeight: theme.typography.fontWeightBold,
+    paddingBottom: "4px",
+  },
+}))
+
+const ControlledInputField = ({ control, name, label, placeholder, type }) => {
+  const classes = useStyles()
+
+  const validateInput = (value) => {
+    if (!value) return false
+    if (value.length < 3) return false
+
+    return true
+  }
```

Comment:
you can get rid of this for now since we will handle validation later, remember, we should only ever have code that is being used in our code base